Webhooks announcements (#1001)
codatio/codat-docs: blog/240216-new-webhook-service-released.md

---

> title: "🚀 Exciting announcement: Introducing our new webhooks service! 🚀"

Reviewer: ```suggestion
title: "🚀 Introducing our new webhooks service 🚀"
```

date: "2024-02-16"
tags: ["Product", "Update"]
authors: dcoplowe
---

> We're thrilled to unveil our latest platform update designed to simplify your development experience: Our brand-new Webhooks Service! 🎉

Reviewer: ```suggestion
We're thrilled to unveil our latest platform update designed to simplify your development experience: our brand-new webhooks service! 🎉
```

<!--truncate-->

> Say goodbye to complex development processes and hello to streamlined efficiency with our best-in-class solution. Here's what you can expect from our new Webhooks Service:

Reviewer: ```suggestion
Say goodbye to complex development processes and hello to streamlined efficiency with our best-in-class solution. Here's what you can expect from our new webhooks service:
```

> ⚙️ **Smart Retries**: No more worrying about missed events or failed deliveries. Our intelligent system automatically retries failed webhook deliveries, ensuring your crucial messages reach their destination without any hassle.

Reviewer: I assume this is a Svix copy... please change not to capitalise both words in bold. (e.g. needs to be Smart retries instead of Smart Retries). will also have to think about rewriting the description - I feel like this is super sales-y and also addressed to us, not our clients.

Author: I've fixed the capitalization. Re. description: it should be sales-y but addressed to our clients so let me know where I've gone wrong.

📝 **Detailed Event Logging**: Gain deep insights into every aspect of your webhook events. With detailed event logging, you'll have a comprehensive overview of delivery attempts, allowing you to resolve any issues as well as view webhook event payloads.

🔄 **Event Replay**: Need to reprocess past events or retry failed event? Our Event Replay feature makes it a breeze to replay both successful and unsuccessful events deliveries, enabling you to recover from any system failures your end.

🧪 **Mocked Events**: Simplify testing and development with our mocked events functionality. Mocked events makes it a breeze to test, debug, and refine your webhook consumer with ease. Simply select the event type you want to trigger and we'll deliver it to your application.

🛡️ **Improved Stability**: Trust in a robust and reliable solution that prioritizes stability. Our Webhooks Service is engineered for resilience, delivering unmatched performance and uptime to support your mission-critical operations.

#### How can I migrate to our new service?

> Checkout out our [migration guide](/using-the-api/webhooks/migration) to see if you can move over and how to request this.

Reviewer: ```suggestion
Check out our [migration guide](/using-the-api/webhooks/migration) to see if you can move over and how to request the move.
```
